lib/render.js

```diff
@@ -132,27 +132,33 @@ function getVis() {
 }
 
 function renderLater() {
-    clearTimeout(reloadTimer);
-    reloadTimer = setTimeout(function() {
-        renderChart();
-    }, 100);
+    _.defer(renderChart);
```

Comment:
can we revert this? importing entire `underscore` for this seems a step in the wrong direction

Reply:
Done